crates/bevy_ui/src/render/mod.rs

```diff
@@ -456,17 +455,27 @@ pub fn extract_uinode_borders(
                 Option<&TargetCamera>,
                 Option<&Parent>,
                 &Style,
-                &BorderColor,
+                Option<&BorderColor>,
+                Option<&Outline>,
             ),
-            Without<ContentSize>,
+            Or<(With<BorderColor>, With<Outline>)>,
         >,
     >,
     node_query: Extract<Query<&Node>>,
 ) {
     let image = AssetId::<Image>::default();
 
-    for (uinode, global_transform, view_visibility, clip, camera, parent, style, border_color) in
-        &uinode_query
+    for (
+        uinode,
+        global_transform,
+        view_visibility,
+        clip,
+        camera,
+        parent,
+        style,
+        maybe_border_color,
+        maybe_outline,
```

Comment:
The other `Option`s are not named `maybe_`. Either remove the `maybe_` or add it to the others as well.